src/main/java/core/basesyntax/service/impl/FileWriterImpl.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileWriter;
+import java.io.BufferedWriter;
+import java.io.IOException;
+
+public class FileWriterImpl implements FileWriter {
+    @Override
+    public void write(String data, String path) {

```

Comment:
No validation checks for the input parameters 'data' and 'path'. An IllegalArgumentException should be thrown if either of these are null or empty.